src/common/transformations/tests/common_optimizations/align_mixed_fp32_fp16_types_test.cpp

```diff
@@ -36,24 +35,21 @@ TEST_F(TransformationTestsF, align_mixed_fp16_fp32_1) {
         pass::Manager manager;
         manager.register_pass<pass::MarkSugraphsToKeepInMixedPrecision>();
         manager.register_pass<pass::AlignMixedFP32FP16Types>();
-        const precisions_array convert_precision_list{{ov::element::f32, ov::element::f16}};
-        type_to_fuse_map empty_type_to_fuse_map = {};
-        manager.register_pass<pass::ConvertPrecision>(convert_precision_list, empty_type_to_fuse_map, true);
         manager.run_passes(model);
     }
 
     {
-        auto input_1 = make_shared<Parameter>(element::f16, Shape{1, 3, 224, 224});
+        auto input_1 = make_shared<Parameter>(element::f32, Shape{1, 3, 224, 224});
         auto convert_to_f32_1 = make_shared<Convert>(input_1, element::f32);
         auto exp_1 = make_shared<Exp>(convert_to_f32_1);
-        auto input_2 = make_shared<Parameter>(element::f16, Shape{1, 3, 224, 224});
```

Comment:
Type rewriting from `f32` to `f16` will be done inside `ConvertPrecision`. 

In `AlignMixedFP32FP16Types` we only insert `Convert`s for the future, so that when element types are changed (precision sensitive nodes remain in `f32` while others are casted to `f16`) inside `ConvertPrecision` there is will be no type inconsistency